package.json

```diff
@@ -21,7 +21,8 @@
   "license": "MIT",
   "dependencies": {
     "follow-redirects": "^1.2.3",
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
+    "node-rsa": "^0.4.2"
```

Comment:
Hi @SidneyAllen , the `node-rsa` dependency should probably be removed also as it is not used

Reply:
@dupski - removed the node-rsa dependency and merged.